activestorage/test/test_helper.rb

```diff
@@ -92,10 +100,14 @@ class User < ActiveRecord::Base
   validates :name, presence: true
 
   has_one_attached :avatar
-  has_one_attached :cover_photo, dependent: false
+  has_one_attached :cover_photo, dependent: false do |attachable|
+    attachable.has_variant :thumbnail, resize: "100x100"
+  end
 
   has_many_attached :highlights
-  has_many_attached :vlogs, dependent: false
+  has_many_attached :vlogs, dependent: false do |attachable|
+    attachable.has_variant :thumbnail, resize: "100x100"
+  end
```

Comment:
Modified the default fixtures to ensure some are configured with defined variants